lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -1,4 +1,5 @@
 [Console]::OutputEncoding = New-Object -typename System.Text.ASCIIEncoding
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]'Ssl3,Tls,Tls11,Tls12'
```

Comment:
I am not sure if older versions of Windows natively support TLS 1.2 or not... but if they do, it should be the only TLS version we use. At the very least we should disable SSLv3, and possibly TLS 1.0.